### ISSUE TITLE
Missing game from ULSC

### ISSUE THREAD

**Issue by ZachWAllen** [American-Soccer-Analysis/asa-shiny-app#114]
It looks like the LA Galaxy II vs OC game from May 29th is missing from the USL Championship fixture list.

**Comment by tyler-richardett** [comment]
Yeah, we had issues with missing data for this game. I'll leave this ticket open and post updates as we work to resolve. Appreciate the note!

**Comment by BentonNewman** [comment]
I would love to see the following data points added for the USLC:

shots on target by team/game
total shots by game
accurate/inaccurate passes by player/team/game
chances/big chances by player/team/game
